### ISSUE TITLE
the effect of the number of personalized tokens

### ISSUE THREAD

**Issue by Tim3s** [HiPer0/HiPer#2]
![training_results_seed200000_step1100](https://github.com/HiPer0/HiPer/assets/80621405/1216e4c3-7ab0-4428-866d-fde9b89d24a8)
I tested this code with lego dozer image from NeRF dataset, and above image is the training result of step 1100 with seed 200000. I set N=20 to get output similar to the source image but it's not even close. I got similar result even if I set N=50. Did I miss something?
The command I used is
`python train.py --pretrained_model_name 'CompVis/stable-diffusion-v1-4' --input_image 'dozer1.png' --target_text 'a broken lego dozer' --source_text 'a lego dozer' --output_dir './dozer1' --n_hiper 20 --emb_learning_rate 5e-3 --emb_train_steps 1200 --seed 200000`

**Comment by Tim3s** [comment]
![a dog sleeping on the beach_seed111111_0](https://github.com/HiPer0/HiPer/assets/80621405/debb911b-f215-4a70-9ad1-dfbe32fbe469)
Tested with default image, still can't get the same result. The prompt for above image is "a dog sleeping on the beach", and commands are just the same as the command given in github, except --n_hiper and test prompt
train:
`python train.py --pretrained_model_name 'CompVis/stable-diffusion-v1-4' --input_image 'dog.png' --target_text 'a sitting dog' --source_text 'a standing dog' --output_dir './standing_dog' --n_hiper 20 --emb_learning_rate 5e-3 --emb_train_steps 1200 --seed 200000`
test:
`python inference.py --pretrained_model_name 'CompVis/stable-diffusion-v1-4' --inference_train_step 1000 --target_txt 'a dog sleeping on the beach' --output_dir './standing_dog' --seed 111111 --image_num 10`
